rllib/algorithms/algorithm_config.py

```diff
@@ -3705,6 +3721,34 @@ def items(self):
         """Shim method to help pretend we are a dict."""
         return self.to_dict().items()
 
+    @property
+    def model_config_dict(self):
+        """Defines the model configuration used.
+
+        This method combines the auto configuration `self._model_auto_keys`
+        defined by an algorithm with the user-defined configuration in
+        `self._model_config_dict`.This configuration dictionary will be used to
+        configure the `RLModule` in the new stack and the `ModelV2` in the old
+        stack.
+
+        Returns:
+            A dictionary with the model configuration.
+        """
+        return self._model_auto_keys | self._model_config_dict
+
+    @property
+    def _model_auto_keys(self) -> Dict[str, Any]:
+        """Defines the main keys for auto-model configuration.
+
+        The dictionary in this property contains the default configuration of an
+        algorithm. Together with the `self._model`, this method will be used to
+        define the configuration sent to the `RLModule`.
+
+        Returns:
+            A dictionary with the main keys for auto-model configuration.
```

Comment:
A dictionary with the automatically included properties/settings of this AlgorithmConfig object into `self.model_config`.